pkg/exporter/ipfix.go

```diff
@@ -12,6 +12,9 @@ import (
 
 var ilog = logrus.WithField("component", "exporter/IPFIXProto")
 
+// TODO: encode also the equivalent of the Protobuf's AgentIP field in a format that is binary-
```

Comment:
For information, in goflow's IPFIX this was called "SamplerAddress": https://github.com/cloudflare/goflow#output-format

In IPFIX spec (https://www.iana.org/assignments/ipfix/ipfix.xhtml) this is `exporterIPv4Address` / `exporterIPv6Address`

Reply:
@jotak the ExportingProcess has a net.Conn, which could be used to get the info. Yes, exporterIPv4(/6)Address seems right though

Reply:
However, the ExportingProcess could be used only if the AgentIPIface is set as "External"